README.md

```diff
@@ -12,3 +12,24 @@ Discovery service doesn't store any data, all data is ephemeral and is kept only
 Node information is expired (if not updated) after 30 minutes.
 Discovery service doesn't see actual node information, it only stores and updates encrypted blobs.
 Discovery data should be submitted encrypted by the client, and service doesn't have the encryption key.
+
+The repo has been split into 3 parts(as they have differnt source code licenses), namely:
+- The current repo
+- [Discovery client](https://github.com/talos-systems/discovery-client), contains the client code to interact with the server
+- [Discovery api](https://github.com/talos-systems/discovery-api/), provides various endpoints to access the server
+
+## Setup 
+
+All of the details to get started are present [in the make file](Makefile), start with `make help`.
+
+
+## Interacting
+
+Once the application is running you can test the grpc functionality on the port 3000 and the http pages in browser on port 3001.
+
+To test the grpc calls, [install grpcurl from any of the ways mentioned here](https://github.com/fullstorydev/grpcurl#installation).
```

Comment:
nit: since this is already a hyperlink, I'd trim the "from any of the ways here" part.  After all, the important part is to install grpcurl; it doesn't matter _how_ they install it, and the hyperlink provides the instructions within.